pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -104,6 +106,38 @@ func (v *vpnShoot) computeResourcesData() (map[string][]byte, error) {
 	var (
 		registry = managedresources.NewRegistry(kubernetes.ShootScheme, kubernetes.ShootCodec, kubernetes.ShootSerializer)
 
+		clusterRole = &rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "system:gardener.cloud:vpn-seed",
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups:     []string{""},
+					Resources:     []string{"services"},
+					ResourceNames: []string{"vpn-shoot"},
+					Verbs:         []string{"get"},
+				},
+			},
+		}
+
+		clusterRoleBinding = &rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        "system:gardener.cloud:vpn-seed",
+				Annotations: map[string]string{resourcesv1alpha1.DeleteOnInvalidUpdate: "true"},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "ClusterRole",
+				Name:     clusterRole.Name,
+			},
+			Subjects: []rbacv1.Subject{
+				{
+					Kind: rbacv1.UserKind,
+					Name: "vpn-seed",
+				},
+			},
+		}
```

Comment:
I see that this was not implemented in the Helm chart, however, both `ClusterRole` and `ClusterRoleBinding` are only needed if `ReversedVPN` is not enabled. If it's enabled then you don't need to add them.